tests/blockchain.rs

```diff
@@ -518,12 +520,14 @@ fn smo_instruction_works() {
         let txid = tx.id();
         let receipts = client.transact(tx);
 
-        let success = receipts.iter().any(|r| match r {
-            Receipt::ScriptResult {
-                result: ScriptExecutionResult::Success,
-                ..
-            } => true,
-            _ => false,
+        let success = receipts.iter().any(|r| {
```

Comment:
with this style, we get an extra level of indentation. Is that an improvement?

Reply:
It's more readable to me, but a personal preference of course. I always code with clippy enabled, so my preferences often match with clippy.